mne/epochs.py

```diff
@@ -308,8 +318,64 @@ def __init__(self, info, data, events, event_id=None, tmin=-0.2, tmax=0.5,
             else:
                 self.drop_log = drop_log
             events = events[selected]
-            if len(np.unique(events[:, 0])) != len(events):
-                raise RuntimeError('Event time samples were not unique')
+            u_evs, u_idxs, counts = np.unique(events[:, 0],
+                                              return_index=True,
+                                              return_counts=True)
+            if len(u_evs) != len(events):
+                msg = 'Multiple event codes for single event times found.'
+                if event_repeated == 'error':
+                    raise RuntimeError('Event time samples were not unique')
+
+                elif event_repeated == 'drop':
+                    logger.info('{} Keeping the first occurrence and '
+                                'dropping all others.'.format(msg))
+                    events = events[u_idxs]
+
+                elif event_repeated == 'merge':
+                    logger.info('{} Creating a new event code to reflect these'
+                                'simultaneous events and updating event_id.'
+                                .format(msg))
+                    new_events = events.copy()
+                    non_u_evs = u_evs[counts > 1]
+                    for ev in non_u_evs:
+
+                        # indices at which the non-unique events happened
+                        idxs = (events[:, 0] == ev).nonzero()[0]
+
+                        # Figure out event_codes prior to events to be merged
+                        prior_codes = events[idxs, 1]
```

Comment:
this needs to be `[idxs, 2]`

Reply:
no, this is to update the second column in the `events` array, which is the event value prior to the current event value (the one of the current row in `events`) ... but I take your comment as such that I should think about my variable names.

Reply:
I wouldn't worry about the second column, no need to merge this one. I would just always use 'drop' behavior (and document it as such). People aren't really supposed to use the second column as it just indicates the transition value (and it makes sense then to keep the value of whichever event was listed first in the events array).

Reply:
but ... doesn't my behavior make sense then? --> if the prior value is consistent across duplicates --> keep it

else: set to np.nan (ignore it)

Reply:
Ahh yes that does make sense. Except for the `np.nan` thing because this is an `int` array, better to use `0`